data/strings/strings.txt

```diff
@@ -13330,7 +13332,7 @@
 
   [login_to_make_edits_visible]
     tags = android,ios
-    en = Log in to OpenStreetMap.org so other users can see the changes you have made.
+    en = Login to upload your changes for other users.
```

Comment:
No. We should teach users about OSM everywhere. Assume that a new user does not know about OSM, and thinks that we (developers) add data on the map. And we should _teach_ _all_ users about OSM's existence. To redirect their map-related issues there, not to us.

Reply:
as a contributor to OSM, i absolutely agree :) the issue is that the login page says OSM too much. maybe the URL could be included in the openstreetmap info card at the top?

Reply:
i'll leave in the link for now, mentioning OSM too much can be fixed another time :)